### PR TITLE
Offshore Geodesy - Add testing for the time.py utility module

### PR DESCRIPTION
**Pull request by madhavmk**
* Add unittest to verify working of custom subclass "TimeUnixJ2000" of astropy's "TimeFromEpoch".
* Linked to issue [uw-ssec/codeuw/issues/29](https://github.com/uw-ssec/codeuw/issues/29)

### CODE REVIEW

**Comment by madhavmk** [comment]
Hello,
I am having a doubt over the implementation of TimeUnixJ2000 subclass. I have raised an issue explaining further [offshore-geodesy/issues/108](https://github.com/uw-ssec/offshore-geodesy/issues/108). Please do let me know if I have misunderstood the definition. Nonetheless, I have added unittests to cover the same.

**Comment by madhavmk** [comment]
This PR now also addresses bug-fix Issue [uw-ssec/offshore-geodesy/issues/108](https://github.com/uw-ssec/offshore-geodesy/issues/108)

File: src/gnatss/utilities/time.py [hunk 1]
```diff
@@ -4,23 +4,20 @@
...
+    ignoring leap seconds.
     """
 
     name = "unix_j2000"
-
     unit = 1.0 / erfa.DAYSEC  # in days (1 day == 86400 seconds)
-
-    epoch_val = "2000-01-01 12:00:00"
-
+    epoch_val = "2000-01-01 11:58:55.816"
```
Comment (lsetiawan) [inline src/gnatss/utilities/time.py:20]: Thanks for making these changes @madhavmk. A little background with this python package is that this is a port from a Fortran program, so I need to ensure the outputs are currently matching. Therefore, it looks like one issue with using the `utc` scale here is that when I call `mytime.iso` I get the incorrect value compared to the Fortran one. For example:

```python
aj2000 = AstroTime(712215465.0, format="unix_j2000")

print(aj2000.iso) # 2022-07-27 17:36:40.816
```

That's what your change would output. However, If I keep the `epoch_val` the way it is and change the scale to be `tt` and do the same thing, I get the correct value of `2022-07-27 17:37:45.000`. If I need utc iso, I can simply do `aj2000.utc.iso`. This is why i suggested this way in the comment: https://github.com/uw-ssec/offshore-geodesy/issues/108#issuecomment-1661005108

File: tests/utilities/test_time.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+from gnatss.utilities.time import AstroTime, erfa, isclose
+
+DATETIME_FORMAT = "%Y-%b-%dT%H:%M:%S.%f"
+
+
+def test_unix_j2000_time_format():
```
Comment (lsetiawan) [inline tests/utilities/test_time.py:6]: Here are 3 data values to test against for the argument above:

```
j2000,expected
712215465.0,2022-07-27T17:37:45.000
712215480.0,2022-07-27T17:38:00.000
712215495.0,2022-07-27T17:38:15.000
```

File: src/gnatss/utilities/time.py (hunk 1 above)
Comment (madhavmk) [reply src/gnatss/utilities/time.py]: You're right. Using `tt` scale matches the fortran library output. I have added a commit updating this PR adding the testcases you provided below.
p.s. Is the fortran module open-sourced? I am interested in checking it out!

**Comment by madhavmk** [comment]
Hello @lsetiawan, I have addressed your PR comments. Using `tt` scale is the correct way to go !

File: src/gnatss/utilities/time.py (hunk 1 above)
Comment (lsetiawan) [reply src/gnatss/utilities/time.py]: >  p.s. Is the fortran module open-sourced? I am interested in checking it out!

The fortran is not open-source unfortunately.

**Review by lsetiawan — APPROVED** [review]

**Pull request merged**